src/mgmt/srv_query.c

```diff
@@ -329,11 +341,10 @@ ds_mgmt_dev_state_query(uuid_t dev_uuid, Mgmt__DevStateResp *resp)
 		goto out;
 	}
 
-	if (dev_info->sdi_state == SMD_DEV_NORMAL)
-		strncpy(resp->dev_state, "NORMAL\n", buflen);
-	else if (dev_info->sdi_state == SMD_DEV_FAULTY)
-		strncpy(resp->dev_state, "FAULTY\n", buflen);
-	else {
+	strncpy(resp->dev_state,
+		smd_state_enum_to_str(dev_info->sdi_state), buflen);
+	if (dev_info->sdi_state != SMD_DEV_NORMAL \
```

Comment:
(style)  Avoid unnecessary line continuations

Reply:
your mom is unnecessary